### ISSUE TITLE
Would like add some snippets to the apex extension.

### ISSUE THREAD

**Issue by Vishal-skywalker** [forcedotcom/salesforcedx-vscode#5107]
Hi SF-vscode team,
I have some suggestions to make about the snippets add to the extension, as a SF dev I feel that these will help in faster development.
* The following piece of code is a few changes I added to the `Apex.json` file in your extension: -
```json
{
  "Is Blank": {
      "prefix": "isb",
      "body": [
          "String.isBlank(${0})"
      ],
      "description": "String isBlank"
  },
  "Is Not Blank": {
      "prefix": "isnb",
      "body": [
          "String.isNotBlank(${0})"
      ],
      "description": "String Not isBlank"
  },
  "local variable": {
      "prefix": "localVariable",
      "body": [
          "${1:Object} ${2:name} = ${3:};"
      ],
      "description": "New static variable"
  },
  "static variable": {
      "prefix": "staticVariable",
      "body": [
          "${1|private,public,global|} static ${2:Object} ${3:name} = ${4:};"
      ],
      "description": "New static variable"
  },
  "variable": {
      "prefix": "variable",
      "body": [
          "${1|private,public,global|} ${2:Object} ${3:name} = ${4:};"
      ],
      "description": "New ariable"
  },
  "static final variable": {
      "prefix": "staticFinalVariable",
      "body": [
          "${1|private,public,global|} static final ${2:Object} ${3:name} = ${4:};"
      ],
      "description": "New static final variable"
  },
  "System Debug 2": {
    "prefix": "sd",
    "body": [
        "System.debug('{$0:data} >>' + {$0:data});"
    ],
    "description": "System.debug"
  }
}
```
* The following is a bug in the `formap` snippet that I fixed :-
```json
{
  "for map": {
      "prefix": "formap",
      "body": [
          "for(${1:String} ${2:key} : ${3:map}.keySet()) {",
          "\t$0",
          "}"
      ],
      "description": "For loop to iterate over a map"
  }
}
```
* This one a much used `test method syntax` :- 
```json
{
  "TestMethod": {
      "prefix": "testMethod",
      "body": [
          "@IsTest",
          "static void ${1:methodName}(){",
          "\t",
          "}"
      ],
      "description": "Test method "
  }
}
```
Thanks!
PS:- Hope the team will find this useful.

**Comment by daphne-sfdc** [comment]
Hi @Vishal-skywalker,

Thank you for your contributions!  Can you make a PR with your code changes?  We'll review and approve your PR, then you can merge those changes into our codebase.

**Comment by daphne-sfdc** [comment]
Closed by merging PR #5108.
